scripts/start_server_and_test

```diff
@@ -14,12 +14,13 @@ START_SERVER_SCRIPT="$1"
 PORT="$2"
 TEST_SCRIPT="$3"
 
-await_port --await-free "$PORT"
+await_port --await-free --timeout 120 "$PORT"
 
 eval "$START_SERVER_SCRIPT &"
-SERVER_PID=$!
 
 function finish {
+  SERVER_PID=`lsof -i -P -n | grep LISTEN | grep ${PORT} | awk '{ print $2 }'`
+  echo Killing server PID=${SERVER_PID}
   kill $SERVER_PID || true
```

Comment:
@atolopko-czi for my own edification, the point of `|| true` here?

Reply:
Assuming it allows the entire script to return success stats code even if the kill fails, though that seems undesirable.  I left as is, since I wasn't entirely sure myself, assuming there was greater wisdom in its creation.